### PR TITLE
Make editor layout more flexible

### PR DESCRIPTION
**Pull request by fregante**
I started this work after seeing this bug, following https://github.com/pixiebrix/pixiebrix-extension/pull/2591:

Screen Shot 7

This happens when using inflexible `width`s and `height`s. The alternative isn't always simple but it results into more forgiving layouts which, even if "not expected", don't look broken:

Screen Shot

To allow for that flexibility, I had to drop the hard-coded `$navHeight` and rework the global layout of the editor, cleaning it up as I went along, as described in:

- https://github.com/pixiebrix/pixiebrix-extension/issues/1234

When it doesn't wrap it's unchanged:

Screen Shot 12

### CODE REVIEW

File: src/devTools/editor/components/ToggleField.tsx [hunk 1]
```diff
@@ -23,11 +23,12 @@ const ToggleField: React.FunctionComponent<{ name: string }> = ({ name }) => {
   const [field, , helpers] = useField(name);
   return (
     <BootstrapSwitchButton
-      size="sm"
+      size="xs"
```
Comment (fregante) [inline src/devTools/editor/components/ToggleField.tsx:26]: Small, semi-related change. I feel that it's difficult to understand it's a switch, it almost looks like an input field.

#### Before

Screen Shot 10

#### After


Screen Shot 9

File: src/devTools/Editor.tsx [hunk 2]
```diff
@@ -140,7 +139,7 @@ const Editor: React.FunctionComponent = () => {
   ]);
 
   return (
-    <div className={styles.root}>
+    <div className="DevToolsContainer">
```
[the remark this answers is not in the document]
Comment (BALEHOK) [reply src/devTools/Editor.tsx:142]: Such usage of classes is hard to trace and may be difficult to support. The CSS class `DevToolsContainer` is defined in `Panel.scss`, used in`Editor.tsx` and the scss file itself if imported in `devtoolsPanel.tsx`.
Ideally, we need to convert `Panel.scss` to scss module.

[the remark this answers is not in the document]
Comment (fregante) [reply src/devTools/Editor.tsx:142]: Agreed, Todd mentioned it in the CSS PR. Out of scope here though, we can open a new issue for that

**Review by BALEHOK — COMMENTED** [review]
In my opinion, things got a little bit messy with the `Editor.tsx` using styles from `Panel.scss` that were updated and not converted to CSS module. I believe this must be resolved, not necessary in this PR though.

**Comment by fregante** [comment]
I re-tested this with more views and it looks good

**Comment by fregante** [comment]
Oops I didn't test a recent addition. Will open a new PR

Screen Shot 7

**Pull request merged**